src/Finder/TestFrameworkFinder.php

```diff
@@ -99,6 +99,7 @@ private function findTestFramework(): string
         $candidates = [
             $this->testFrameworkName,
             $this->testFrameworkName . '.phar',
+            $this->testFrameworkName . '.bat', // always looking for .bat for testing with .bat not on Windows
```

Comment:
Now we are looking for the `.bat` version twice.

Reply:
We could be like:
```
if ('\\' == \DIRECTORY_SEPARATOR) {
    array_unshift($candidates, $this->testFrameworkName . '.bat');
} else {
    $candidates[] = $this->testFrameworkName . '.bat';
}
```
Do you think this would be better? I'm not a big fan of `else` it it can be avoided.

Reply:
IMO, in this case `else` is better than finding 2 times `.bat` file

Reply:
OK, done.